processor/src/services/converters/helper.converter.ts

```diff
@@ -12,7 +12,7 @@ import {
   getCtSessionIdFromContext,
   getProcessorUrlFromContext,
 } from '../../libs/fastify/context/context';
-import { NormalizedShipping } from '@commercetools/connect-payments-sdk/dist/commercetools/types/cart.type';
+import { NormalizedShipping } from '@commercetools/connect-payments-sdk';
```

Comment:
Should probably add it to the list of imports on line 3-9

Reply:
Ups.. missed that